packages/rollup-contracts/contracts/L2ToL1MessageReceiver.sol

```diff
@@ -0,0 +1,58 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* Internal Imports */
+import {DataTypes as dt} from "./DataTypes.sol";
+
+contract L2ToL1MessageReceiver {
+    event L2ToL1MessageEnqueued(
+        address ovmSender,
+        bytes callData,
+        uint nonce
+    );
+
+    struct EnqueuedL2ToL1Message {
+        dt.L2ToL1Message message;
+        uint blockEnqueued;
+    }
+    
+    address public trustedSequencer;
```

Comment:
How do you feel about removing the word `trusted`? My rationale is that it's implied and may draw unnecessary negative attention.